controls/raddiagram/howto/custom_connectors.md

```diff
@@ -0,0 +1,127 @@
+---
+title: How to populate RadDiagramShape with custom connectors in MVVM scenario
+page_title: How to populate RadDiagramShape with custom connectors in MVVM scenario
+description: How to populate RadDiagramShape with custom connectors in MVVM scenario.
+slug: raddiagram-howto-create-populate-raddiagramshape-with-custom-connectors-in-mvvm-scenario
+tags: create,custom,shape,connectors,mvvm,graphsource
+published: True
+position: 18
+---
+
+# How to populate RadDiagramShape with custom connectors in MVVM scenario
+
+This tutorial will guide you through the task of populating RadDiagramShape with custom connectors in MVVM scenario.
+
+To populate the __RadDiagramShape__ with custom connectors in MVVM scenario you can create an attached property. Then bind the Value of this property to a collection from your ViewModel. In the attached property you have access to the shape and its Connectors collection. Using this you can add new connectors to the collection. 
+
+To implement this scenario you can create a collection which holds the custom connectors in your ViewModel.
+
+#### __[C#] Example 1: Creating collection with custom connectors__
+{{region raddiagram-howto-create-populate-raddiagramshape-with-custom-connectors-in-mvvm-scenario-0}}
```

Comment:
If you change the title and the slug you can consider also changing the regions.